gnosis/gp-v2-services
# Set X-Partner header on Paraswap queries

Context: https://gnosisinc.slack.com/archives/C0203Q9R4JJ/p1630337209009700

### Test Plan
- Install proxyman and add certificate to local keychain to intercept HTTPs requests
- export `https_proxy=<proxyman listening port>`
- run solver on mainnet, watch paraswap API requests, see header being set.
